src/redux/reducers/network.js

```diff
@@ -49,6 +49,19 @@ export default function networkReducer(state = initialState, action) {
       return Object.assign({}, state, { pending });
     }
 
+    case Actions.INVLIDATE_ERROR: {
+      const id = action.id;
```

Comment:
thanks, fixed that

On Wed, Apr 6, 2016 at 11:28 AM, Chris Harris <notifications@github.com>
wrote:

> In src/redux/reducers/network.js
> <https://github.com/Kitware/HPCCloud/pull/323#discussion_r58747051>:
>
> > @@ -49,6 +49,19 @@ export default function networkReducer(state = initialState, action) {
> >        return Object.assign({}, state, { pending });
> >      }
> >
> > +    case Actions.INVLIDATE_ERROR: {
> > +      const id = action.id;
>
> INVLIDATE_ERROR => INVALIDATE_ERROR?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Kitware/HPCCloud/pull/323/files/4f272feea57defada9ec8eb5533ba80b88ab17cd#r58747051>
>



-- 
- Tristan Wright

*R&D Engineer*
*Kitware Inc.*